src/test/resources/generatedFiles/deployment.tmpl

```diff
@@ -0,0 +1,44 @@
+# This file is generated by tools/run/run.go
```

Comment:
Could we download this file dynamically since it will change over time. (e.g. metadata versions each release)